stream-chat-android-ui-components/src/main/res/values/attrs.xml

```diff
@@ -225,5 +288,31 @@
         <attr name="streamUiScrollButtonRippleColor" format="color|reference" />
         <attr name="streamUiScrollButtonBadgeColor" format="color|reference" />
         <attr name="streamUiScrollButtonIcon" format="reference" />
+
+        <attr name="streamUiLoadMoreThreshold" format="integer" />
+        <attr name="streamUiMessagesEmptyStateLabelText" format="string" />
+        <attr name="streamUiNewMessagesBehaviour" format="enum">
+            <enum name="scroll_to_bottom" value="0" />
+            <enum name="count_new_messages" value="1" />
+        </attr>
+
+        <attr name="streamUiReplyOptionMessage" format="string" />
+        <attr name="streamUiReplyOptionIcon" format="reference" />
+        <attr name="streamUiThreadReplyOptionMessage" format="string" />
+        <attr name="streamUiThreadReplyOptionIcon" format="reference" />
+        <attr name="streamUiCopyOptionMessage" format="string" />
+        <attr name="streamUiCopyOptionIcon" format="reference" />
+        <attr name="streamUiMuteOptionMessage" format="string" />
+        <attr name="streamUiMuteOptionIcon" format="reference" />
+        <attr name="streamUiBlockOptionMessage" format="string" />
+        <attr name="streamUiBlockOptionIcon" format="reference" />
+        <attr name="streamUiDeleteOptionMessage" format="string" />
+        <attr name="streamUiDeleteOptionIcon" format="reference" />
+        <attr name="streamUiMessageOptionIconColor" format="color" />
```

Comment:
These were previously declared on the old styleable, but they belong to the new one